.github/workflows/ci.yaml

```
@@ -45,6 +47,12 @@ jobs:
           path: build/brimcap-*.zip
       - if: startsWith(github.event.ref, 'refs/tags/')
         uses: svenstaro/upload-release-action@1.1.0
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.ref }}
+          file_glob: true
+          file: build/brimcap-*.zip
```

`file` precedes `file_glob` in our other workflows.
```suggestion
          file: build/brimcap-*.zip
          file_glob: true
```